front/src/modules/ui/board/components/BoardColumn.tsx

```diff
@@ -72,7 +51,7 @@ const StyledNumChildren = styled.div`
   width: 16px;
 `;
 
-type OwnProps = {
+export type BoardColumnProps = {
```

Comment:
Props should be exposed as part of the component API and have the component name in their name